tools/SeeDot/seedot/arduino/compileConfig.h

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+// The datatype of the fixed-point representation is specified below.
+#define INT16
+#define XFLOAT
```

Comment:
Ideally, these should be compile time flags, no? I don't see why these need to be defined in a separate file. I think it's better to document these instead.